app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -108,12 +107,23 @@ func NewLoadBalancerSyncer(lbName string, clients map[string]kubeclient.Interfac
 
 // CreateLoadBalancer creates the GCP resources necessary for an L7 GCP load balancer corresponding to the given ingress.
 // clusters is the list of clusters that this load balancer is spread to.
-func (l *LoadBalancerSyncer) CreateLoadBalancer(ing *v1beta1.Ingress, forceUpdate bool, clusters []string) error {
+func (l *LoadBalancerSyncer) CreateLoadBalancer(ing *v1beta1.Ingress, forceUpdate, validate bool, clusters []string) error {
 	client, cErr := getAnyClient(l.clients)
 	if cErr != nil {
 		// No point in continuing without a client.
 		return cErr
 	}
+
+	if validate {
+		if err := validations.Validate(l.clients, ing); err != nil {
+			fmt.Println("error: validation failed:", err)
```

Comment:
This will result in printing the error twice? The caller also prints the error

Reply:
good catch. I often forget/don't worry about errors being printed multiple times. I fixed up 2 other spots where we were printing it extra.

Reply:
Great! Thanks for fixing those other places as well :)